### DISCUSSION TITLE
Combine fuel cell model with Pybamm

### DISCUSSION THREAD

**Discussion by ngvhien** [pybamm-team/PyBaMM#2182] [Ideas]
Hi everyone
I am having an idea to combine fuel cell and battery when using these 2 models. Can python do it like in matlab/simulink?
Hope everybody help please.
Best regards.

**Comment by rtimms** [comment]
Can you provide more details on what you are trying to do?

**Reply by ngvhien** [reply]
> Can you provide more details on what you are trying to do?

Yes, Currently I've got a Fuel cell model with opem model. And I also use the battery model from pybamm. So I can connect them to work like in the real model, which is simulated in simulink.
https://drive.google.com/file/d/16kAxx3N5MrfXInsbiLxsNp3g5n0kAz3R/view?usp=sharing
Thanks so much
